pages/index.html

```diff
@@ -4,9 +4,29 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>✨Kassidy's Portfolio Site✨</title>
```

Comment:
The index.html page generally lives at the top level of the project directory (not within a subdirectory). When we visit a website (e.g., [www.github.com](http://www.github.com/)) when no page is part of the URL, the web server will look for one of a few default file names (index.html is usually among them) and serve the first one it finds.

While a web server can be configured to serve a default file from elsewhere (such as under the pages folder) it's often easier to have the index.html live in the site root, and have the other pages stored under the pages folder. This affects the paths required for locating style files and images, or other pages in anchor tags.